core/ecschema-rpc/common/src/ECSchemaRpcLocater.ts

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { ISchemaLocater, Schema, SchemaContext, SchemaKey, SchemaMatchType, SchemaProps } from "@bentley/ecschema-metadata";
+import { IModelRpcProps } from "@bentley/imodeljs-common";
+import { ECSchemaRpcInterface } from "./ECSchemaRpcInterface";
+
+/**
+ * Defines a schema locater that retrieves schemas using an RPC interface.
+ * @alpha
+ */
+export class ECSchemaRpcLocater implements ISchemaLocater {
+    public token: IModelRpcProps;
+    
+    constructor(token: IModelRpcProps) { this.token = token; }
+
+    public async getSchema<T extends Schema>(schemaKey: SchemaKey, matchType: SchemaMatchType, context: SchemaContext): Promise<T | undefined> {
```

Comment:
`@public` methods lack documentation here, but they are documented in base class. Mark them `@internal`?

Reply:
Colin advised that I added documentation instead, so I pushed a commit that did so.

Reply:
@ColinKerr Wouldn't it be preferable to document the interface's methods on the interface? Do you find it awkward that the documentation on this implementation is not the same as that on the interface?

Reply:
@hl662 gotta rerun extract-api after changing documentation.

Reply:
This implementation of the interface is a bit of an odd implementation over the other ones because it is reaching out over the rpc.  I think some specific documentation is needed here.  These docs aren't great but this is just an initial alpha push so we can get some testing done integrating this up the stack.

So I'm OK with the docs that are there now and we'll work on them as we get further along.

Reply:
Famous last words...